scripts/scripts.js

```diff
@@ -318,6 +334,9 @@ function autolinkModals(element) {
       openModal(origin.href);
     }
   });
+  const { openOnTrigger } = await import(`${window.hlx.codeBasePath}/blocks/form/trigger.js`);
+  const { openModal } = await import(`${window.hlx.codeBasePath}/blocks/modal/modal.js`);
+  openOnTrigger(fetchTriggerConfig(), openModal);
```

Comment:
Why above code is part of autoLinkModals? we can make it all part of delayed and also shift delayed logic in trigger